cfgov/v1/jinja2/v1/layouts/base.html

```diff
@@ -56,7 +56,13 @@
     <meta name="description"
           content="
             {%- block desc -%}
-                {{- meta_description if page else '' -}}
+                {% if page %}
+                {{- meta_description -}}
+                {% elif manual_desc %}
+                {{- manual_desc -}}
+                {% else %}
+                {{- '' -}}
+                {%- endif -%}
```

Comment:
You can totally revert these changes since you're now overriding the `desc` block